app/code/Magento/Catalog/Api/GetProductTypeBySkuInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Api;
+
+/**
+ * Get product type id by product sku.
+ *
+ * @api
+ */
+interface GetProductTypeBySkuInterface
+{
+    /**
+     * Returns product type id by product sku.
+     *
+     * @param string $sku
+     * @return string|null
```

Comment:
We need throw `\Magento\Framework\Exception\InputException` if sku does not exist
Cover this case with integration tests

Reply:
@naydav done.